doc/cephadm/troubleshooting.rst

```diff
@@ -161,3 +161,15 @@ This means that you must run a command of this form::
   ceph config set mon public_network <mon_network>
 
 For more detail on operations of this kind, see :ref:`deploy_additional_monitors`
+
+Accessing the admin socket
+--------------------------
+
+Each Ceph daemon provides an admin socket bypassing the
+MONs (See :ref:`rados-monitoring-using-admin-socket`).
+
+As the daemons run within containers, we first have to enter the container on
+the host before accessing the admin socket::
```

Comment:
```suggestion
To access the admin socket, first enter the daemon container on the host::
```
In addition to the suggestion from @asettle ... also drop the phrase `As daemons run within containers` ?